rtgui/paramsedited.cc

```diff
@@ -3450,6 +3460,14 @@ void ParamsEdited::combine(rtengine::procparams::ProcParams& toEdit, const rteng
         toEdit.distortion.amount = dontforceSet && options.baBehav[ADDSET_DIST_AMOUNT] ? toEdit.distortion.amount + mods.distortion.amount : mods.distortion.amount;
     }
 
+    if (distortion.defish) {
+        toEdit.distortion.defish = mods.distortion.defish;
+    }
+
+    if (distortion.focal_length != DistortionParams::DEFAULT_FOCAL_LENGTH) {
+        toEdit.distortion.focal_length = dontforceSet && options.baBehav[ADDSET_DIST_FOCAL_LENGTH] ? toEdit.distortion.focal_length + mods.distortion.focal_length : mods.distortion.focal_length;
+    }
```

Comment:
Similar code is needed for the scale parameter.

Reply:
I thought the similar code is already present in lines 3447-3450
```
    if (commonTrans.scale) {
        toEdit.commonTrans.scale = mods.commonTrans.scale;
    }
```

Reply:
Oh, I selected too much code for this comment. I mean the scale should handle the add/set behavior like what's done to the focal length here.